Sunrise/Helpers/Dictionary+LocalizedString.swift

```diff
@@ -0,0 +1,23 @@
+//
+// Copyright (c) 2016 Commercetools. All rights reserved.
+//
+
+import Foundation
+
+extension Dictionary where Key: StringLiteralConvertible, Value: StringLiteralConvertible {
+
+    var localizedString: Value? {
+        let currentLocaleIdentifier = NSLocale.currentLocale().localeIdentifier
+
+        if let key = currentLocaleIdentifier as? Key, localizedString = self[key] {
+            return localizedString
+
+        } else if let key = NSLocale.componentsFromLocaleIdentifier(currentLocaleIdentifier)["kCFLocaleLanguageCodeKey"] as? Key, localizedString = self[key] {
```

Comment:
We can use the constant `NSLocaleLanguageCode` here instead of the String `"kCFLocaleLanguageCodeKey"`.